cumulusci/utils.py

```diff
@@ -132,10 +132,8 @@ def remove_xml_element(name, tree):
     return tree
 
 
-def download_extract_zip(url, target=None, subfolder=None, headers=None):
```

Comment:
@1handclapping it's probably okay, but it would be safer if we keep the `headers` param; I haven't reviewed all our projects to make sure this util isn't used somewhere else.